### PR TITLE
Added old homepage and login page

### PR DESCRIPTION
**Pull request by RJaliliSTFC**
Copied old HTML for the homepage and login page
Moved the images one folder up because it was doubled Modified cloud.py to direct to the pages

### CODE REVIEW

File: cloud.py [hunk 1]
```diff
@@ -13,20 +13,15 @@ def terms():
 
 @app.route("/sla")
 def sla():
-    return "Service Level Agreement Page <h1>SLA<h1>"
+    return render_template("sla.html")
 
 # @app.route("/faqs")
 # def faqs():
 #     return "Frequently Asked Questions Page <h1>FAQ<h1>"
```
Comment (meoflynn) [inline cloud.py:20]: Are these to be added in a future PR?

File: cloud.py [hunk 2]
```diff
@@ -13,20 +13,15 @@ def terms():
...
-#         session["user"] = user
-#         return redirect(url_for("user"))
-#     else:
-#         return render_template("loginpage.html")
+@app.route("/login", methods=["POST", "GET"])
+def login():
+        return render_template("login.html")
 
 # @app.route("/user")
 # def user():
```
Comment (meoflynn) [inline cloud.py:27]: ^^

**Review by meoflynn — COMMENTED** [review]
Looks good, I just wanted to check whether some of the commented out lines in `cloud.py` are to be added to a future PR?

File: cloud.py (hunk 1 above)
Comment (RJaliliSTFC) [reply cloud.py:20]: Yeh, another page that I am still reviewing to put into the structure

File: cloud.py (hunk 2 above)
Comment (RJaliliSTFC) [reply cloud.py:27]: This one was being used for debugging, I'll remove it in the next pr

**Review by meoflynn — APPROVED** [review]

**Pull request merged**